pkg/i2gw/providers/nginx/nginx.go

```diff
@@ -25,10 +25,19 @@ import (
 	"github.com/kubernetes-sigs/ingress2gateway/pkg/i2gw/intermediate"
 )
 
-const Name = "nginx"
+const (
+	Name = "nginx"
+
+	GlobalConfigurationFlag = "global-configuration"
+)
 
 func init() {
 	i2gw.ProviderConstructorByName[Name] = NewProvider
+
+	i2gw.RegisterProviderSpecificFlag(Name, i2gw.ProviderSpecificFlag{
+		Name:        GlobalConfigurationFlag,
+		Description: "Name of NIC GlobalConfiguration resource.",
```

Comment:
```suggestion
		Description: "Namespace and Name of NIC GlobalConfiguration resource.",
```
When defining a GlobalConfiguration resource, you need to explicitly set the namespace/name of the resource.
https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#-global-configuration-string